lib/ip_access_control.ex

```diff
@@ -206,8 +207,8 @@ defmodule IpAccessControl do
   def parse_allow_list(list) do
     Enum.map(list, fn item ->
       case item do
-        %RemoteIp.Block{} -> item
-        _ -> RemoteIp.Block.parse!(item)
+        %BitwiseIp.Block{} -> item
+        _ -> BitwiseIp.Block.parse!(item)
```

Comment:
It would be nice if the handling of `BitwiseIp.Blocks.parse` handled this case, too so that I don’t have to make this distinction myself. I think it amounts to roughly the same.

Reply:
I think I'm struggling to imagine when already-parsed blocks would be passed in as an option. Is there a particular scenario? I guess maybe if you did a `BitwiseIp.Blocks.optimize/1` by hand and passed those results into the plug...but then you already have the `optimize/1` call in `IpAccessControl.Options` anyway. 🧐

Reply:
For me, I think that this is a bit of a "belt and suspenders" approach that I’m taking because I can’t be sure what the end-user of the library is going to be passing in.